### PR TITLE
Update area-owners.md

### PR DESCRIPTION
**Pull request by amcasey**
Removed area-runtime, titled notes column, added notes, fixed whitespace to make markdown readable.

### CODE REVIEW

File: docs/area-owners.md [hunk 1]
```diff
@@ -4,30 +4,29 @@ The below table lists all the `area-`labels used in the **dotnet/aspnetcore** re
...
+| **[area-routing](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-routing)**                   |                                                                                               | [@mkArtakMSFT](https://github.com/mkArtakMSFT)           |                         |
+| **[area-security](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-security)**                 |                                                                                               | [@blowdart](https://github.com/blowdart)                 |                         |
+| **[area-signalr](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-signalr)**                   | [@BrennanConroy](https://github.com/BrennanConroy)                                            | [@adityamandaleeka](https://github.com/adityamandaleeka) | Home repo: https://github.com/SignalR/SignalR/ |
```
Comment (adityamandaleeka) [inline docs/area-owners.md:27]: SignalR/SignalR is the repo for classic (non-core) SignalR. I don't think there's value in linking that from here. cc @BrennanConroy

File: docs/area-owners.md [hunk 2]
```diff
@@ -4,30 +4,29 @@ The below table lists all the `area-`labels used in the **dotnet/aspnetcore** re
...
+| **[area-ui-rendering](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-ui-rendering)**         |                                                                                               | [@mkArtakMSFT](https://github.com/mkArtakMSFT)           |                         |
+| **[area-web-frameworks](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-web-frameworks)**     | [@captainsafia](https://github.com/captainsafia) [@mitchdenny](https://github.com/mitchdenny) | [@adityamandaleeka](https://github.com/adityamandaleeka) | Deprecated              |
```
Comment (adityamandaleeka) [inline docs/area-owners.md:29]: Maybe clarify that the _label_ is deprecated?

File: docs/area-owners.md (hunk 1 above)
Comment (amcasey) [reply docs/area-owners.md:27]: Is there a better link or should I just drop it?

File: docs/area-owners.md (hunk 2 above)
Comment (amcasey) [reply docs/area-owners.md:29]: I'm curious what alternative interpretation you have in mind.

Comment (adityamandaleeka) [reply docs/area-owners.md:29]: The very notion of web frameworks 😆

Comment (amcasey) [reply docs/area-owners.md:29]:
```suggestion
| **[area-web-frameworks](https://github.com/dotnet/aspnetcore/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-web-frameworks)**     | [@captainsafia](https://github.com/captainsafia) [@mitchdenny](https://github.com/mitchdenny) | [@adityamandaleeka](https://github.com/adityamandaleeka) | Label is deprecated              |
```

**Review by adityamandaleeka — APPROVED** [review]

**Pull request merged**